handlers/whatsapp/whatsapp.go

```diff
@@ -1022,6 +1034,15 @@ func buildWhatsAppHeaders(channel courier.Channel) http.Header {
 	return header
 }
 
+func hasTiersError(payload mtErrorPayload) bool {
+	for _, err := range payload.Errors {
+		if err.Code == 471 && err.Title == "Spam rate limit hit" {
```

Comment:
is it important to check error title? Just wondering if that can change

Reply:
I can remove that